backend/src/main/java/de/cae/XYFleet/ressource/BookingResource.java

```diff
@@ -8,50 +8,42 @@
 import static de.cae.XYFleet.authentication.XYAuthorizer.ROLE_USER;
 import static org.jooq.codegen.XYFleet.Tables.BOOKINGS;
 
-public class BookingResource extends XYServerResource {
-    private int identifier;
-
+public class BookingResource extends EntryResource {
+    @Override
     @Get("txt")
     public String toString() throws ResourceException {
         checkInRole(ROLE_SECURITY);
-        //SELECT * FROM bookings where id = {identifier}
+        //SELECT * FROM bookings where id = {Identifier}
         BookingsRecord result = dslContext.fetchOne(BOOKINGS, BOOKINGS.ID.eq(identifier));
 
         if (result == null) throw new ResourceException(404, "not found");
         return result.formatJSON(jSONFormat);
     }
-
     @Override
-    protected void doInit() throws ResourceException {
-        super.doInit();
-        try {
-            identifier = Integer.parseInt(getAttribute("identifier"));
-        } catch (NumberFormatException e) {
-            identifier = -1;
-        }
-    }
-
     @Delete()
-    public String deleteBooking() throws ResourceException {
+    public String deleteEntry() throws ResourceException {
         checkInRole(ROLE_USER);
 
         String result = this.toString();
 
 
         if (Integer.parseInt(getClientInfo().getUser().getIdentifier()) != identifier && !isInRole("admin"))
             throw new ResourceException(403);
-        //DELETE bookings where id = {identifier}
+        //DELETE bookings where id = {Identifier}
         dslContext.delete(BOOKINGS).where(BOOKINGS.ID.eq(identifier)).execute();
         return result;
     }
-
-    @Post()
-    public void createBooking() {
+    @Override
+    @Put()
+    public String createEntity() {
         checkInRole(ROLE_USER);
+        return null;
```

Comment:
I'll just assume you are aware of this everywhere else

Reply:
well yes